Cliqz/HomePanel/VPN/VPNCountrySelectionController.swift

```diff
@@ -0,0 +1,280 @@
+//
+//  VPNCountrySelectionController.swift
+//  VPNViews
+//
+//  Created by Tim Palade on 10/26/18.
+//  Copyright © 2018 Cliqz. All rights reserved.
+//
+#if PAID
+import UIKit
+
+protocol VPNCountrySelectionDelegate: class {
+    func didSelectCountry(country: VPNCountry)
+}
+
+/// Allows the user to select from a list of VPNCountry instances, reports back to the `delegate`.
+class VPNCountrySelectionController: UIViewController {
+    // MARK: Delegation
+    weak var delegate: VPNCountrySelectionDelegate? = nil
+
+    // MARK: Properties
+    private let tableView = UITableView()
+    private let backgroundView = BrowserGradientView()
+    private var countries = VPNEndPointManager.shared.getAvailableCountries()
+
+    // MARK: View LifeCycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+
+        tableView.refreshControl = UIRefreshControl()
+        tableView.refreshControl?.tintColor = UIColor.white
+        tableView.register(VPNCountrySelectionCountryCell.self, forCellReuseIdentifier: VPNCountrySelectionCountryCell.reuseIdentifier)
+
+        self.navigationItem.title = NSLocalizedString("Available VPN Locations", tableName: "Lumen", comment: "[VPN] vpn locations")
+
+        NotificationCenter.default.addObserver(self, selector: #selector(receiveUpdatedData),
```

Comment:
observer not removed, can lead to calling the selector when object is dead. 
deinit {
        NotificationCenter.default.removeObserver(self)
    }